src/parser/jobs/ast/modules/Draw.tsx

```diff
@@ -46,53 +50,62 @@ export default class Draw extends Module {
 	private draws = 0
 	private drawDrift = 0
 	private drawTotalDrift = 0
-
 	private plays = 0
-	private prepullPrepped = false
-
 	private lastSleeveTimestamp = 0
-	private sleeveUses = 0
+	private sleevesUsed = 0
 	private sleeveDrift = 0
 	private sleeveTotalDrift = 0
 
+	private prepullPrepped = false
 	private prepullSleeve = false
 
-	private PLAY: number[] = []
-	private ARCANA_STATUSES: number[] = []
-
-	protected override init() {
-
-		PLAY.forEach(actionKey => {
-			this.PLAY.push(this.data.actions[actionKey].id)
-		})
-
-		ARCANA_STATUSES.forEach(statusKey => {
-			this.ARCANA_STATUSES.push(this.data.statuses[statusKey].id)
-		})
-
-		this.addEventHook('cast', {abilityId: this.data.actions.DRAW.id, by: 'player'}, this.onDraw)
-		this.addEventHook('cast', {abilityId: this.data.actions.SLEEVE_DRAW.id, by: 'player'}, this.onSleeveDraw)
-		this.addEventHook('cast', {abilityId: this.PLAY, by: 'player'}, this.onPlay)
-
-		this.addEventHook('applybuff', {abilityId: this.data.statuses.SLEEVE_DRAW.id, by: 'player'}, this.onSleeveBuff)
-		this.addEventHook('applybuff', {abilityId: this.ARCANA_STATUSES, by: 'player'}, this.onPlayBuff)
+	private play: Array<Action['id']> = []
```

Comment:
yes, there are multiple actions that come with play which depend on card type

Reply:
My point was more to call it plays rather than play, but the name is already used so hm. Not a big deal functionally, just a bit spicy for code legibility.

Reply:
yeah I definitely didn't get that from the original message
anywho I updated the name to playAction for legibility